manimpango/cmanimpango.pyx

```diff
@@ -256,3 +256,88 @@ def text2svg(
     g_object_unref(layout)
     return file_name
 
+class MarkupUtils:
+    @staticmethod
+    def text2svg(
+        text:str,
+        font:str,
+        slant: str,
+        weight: str,
+        size:int,
+        line_spacing:int,
+        disable_liga:bool,
+        file_name:str,
+        START_X:int,
+        START_Y:int,
+        width:int,
+        height:int,
```

Comment:
```suggestion
        text: str,
        font: str,
        slant: str,
        weight: str,
        size: int,
        line_spacing: int,
        disable_liga: bool,
        file_name: str,
        START_X: int,
        START_Y: int,
        width: int,
        height: int,
```

Reply:
Thanks. My editor refused to run `black` and I did not think to do it manually.

Reply:
Black doesn't work for `.pyx` files sadly.

Reply:
I found that out just now, while I was looking for a way to change the editor's config :)